pdf/renderer.go

```diff
@@ -1002,6 +1015,45 @@ func (w *pdfPageWriter) embedImage(img image.Image, enc canvas.ImageEncoding) pd
 	return name
 }
 
+func (w *pdfPageWriter) embedJpeg(img canvas.JPEGImage) pdfName {
```

Comment:
Please capitalize JPEG

Reply:
Removed in favor of `imageStream` / `jpegStream`